src/Components/Scenes/Governance/sideBar.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
```

Comment:
Title of the file name should be `Sidebar.tsx` instead of `sideBar.tsx`

Reply:
Updated on my next PR